content/docs/2.12/concepts/scaling-deployments.md

```diff
@@ -66,6 +66,17 @@ spec:
           - type: Percent
             value: 100
             periodSeconds: 15
+    complexScalingLogic:                                    # Optional. Section to specify complex scaling logic
+      formula: {formula-for-fetched-metrics}                # Optional. Formula for calculation
+      target: {target-value-to-scale-on}                    # Optional. If metrics are anyhow composed together
+      externalCalculators:                                  # Optional. Section for user defined grpc calculation servers
```

Comment:
As above, something like `externalModifiers`?

Reply:
`externalModifiers` does sound pretty good. combining these two would then be 
```
modifiers:
 externalModifiers:
  - name: ...
 ```
what about just leaving the servers as `external` as they would be already in `modifiers` field?

Reply:
Ive opened https://github.com/kedacore/keda/pull/4583 for review. Im guessing this will be discussed there